components/bms_worker/src/CAN/CANIO-tx.c

```diff
@@ -63,28 +64,38 @@ static const packTable_S* packNextMessage(const packTable_S* packTable,
                                           CAN_data_T*        message,
                                           uint8_t*           nextCounter);
 
-static bool MSG_pack_BMS_100Hz(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_100Hz1(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_100Hz2(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_100Hz3(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_100Hz_Critical(CAN_data_T* message, const uint8_t counter);
 static bool MSG_pack_BMS_10Hz(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_1Hz(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_SOC_Voltage_Temp(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Cell_Temp_2_to_10(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Cell_Temp_11_to_19(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Cell_Voltage_0_to_5(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Cell_Voltage_6_to_11(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Cell_Voltage_12_to_15(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Temperatures_and_Humidity(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_1Hz_Fans(CAN_data_T* message, const uint8_t counter);
 
 static const packTable_S MSG_packTable_100Hz[] = {
-    //{ &MSG_pack_BMS_100Hz, 0x100, 8U },
-    //{ &MSG_pack_BMS_100Hz1, 0x110, 8U },
+    { &MSG_pack_BMS_100Hz_Critical, 0x100, 8U },
 };
 static const packTable_S MSG_packTable_10Hz[] = {
     { &MSG_pack_BMS_10Hz, 0x10, 8U },
 };
 static const packTable_S MSG_packTable_1Hz[] = {
-    { &MSG_pack_BMS_1Hz, 0x1, 8U },
-    { &MSG_pack_BMS_100Hz, 0x100, 8U },
-    { &MSG_pack_BMS_100Hz1, 0x110, 8U },
-    { &MSG_pack_BMS_100Hz2, 0x700, 8U },
-    { &MSG_pack_BMS_100Hz3, 0x740, 8U },
+    { &MSG_pack_BMS_1Hz_SOC_Voltage_Temp, 0x700, 8U },
+    { &MSG_pack_BMS_1Hz_Cell_Temp_2_to_10, 0x710, 8U },
+    { &MSG_pack_BMS_1Hz_Cell_Temp_11_to_19, 0x720, 8U },
+    { &MSG_pack_BMS_1Hz_Cell_Voltage_0_to_5, 0x730, 8U },
+    { &MSG_pack_BMS_1Hz_Cell_Voltage_6_to_11, 0x740, 8U },
+    { &MSG_pack_BMS_1Hz_Cell_Voltage_12_to_15, 0x750, 8U },
+    { &MSG_pack_BMS_1Hz_Temperatures_and_Humidity, 0x760, 8U },
+    { &MSG_pack_BMS_1Hz_Fans, 0x770, 8U },
```

Comment:
Not all of these messages are as small as they can be. if the message uses 48 bits, it should be a 6 byte message